src/api.py

```
@@ -656,11 +720,216 @@ async def process_evaluation_background(evaluation_id: int):
                 )
             )
 
+async def process_evaluation_run_background(run_id: int):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)